automation/test.sh

```diff
@@ -43,14 +43,54 @@ else
 fi
 
 export KUBEVIRT_NUM_NODES=2
-export NFS_WINDOWS_DIR=${NFS_WINDOWS_DIR:-/home/nfs/images/windows2016}
+export WINDOWS_NFS_DIR=${WINDOWS_NFS_DIR:-/var/lib/stdci/shared/images/windows2016}
+export WINDOWS_LOCK_PATH=${WINDOWS_LOCK_PATH:-/tmp/download_windows_image.lock}
```

Comment:
/tmp/ is not shared among the containers, why not put the lock in the shared dir? Or something exports WINDOWS_LOCK_PATH before this code runs? (if so, I couldn't find where...)

Reply:
not thought about it, thx will fix it